src/Cli/dotnet/commands/dotnet-workload/install/IWorkloadUnitInstaller.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Workloads.Workload.Install
+{
+    internal abstract class IWorkloadUnitInstaller : IWorkloadInstaller
```

Comment:
Should this be named WorkloadUnitInstallerBase since it's abstract and not an interface?